indexes.sql

```diff
@@ -0,0 +1,64 @@
+-- These are optional but suggested indexes for rendering OpenStreetMap Carto
+-- with a full planet database.
+
+-- Ideally the indexes would be created with fillfactor=100 when updates are not
+-- consumed, but this does the simpler case of being used where minutely updates
+-- are consumed.
+
+-- A goal with the indexes is to have them general-purpose enough to not need
+-- frequent changing with stylesheet changes, but to be usable with many
+-- versions, and potentially other styles.
+
+-- The point tables are small, but so are the partial indexes
+CREATE INDEX planet_osm_point_place_low
+  ON planet_osm_point USING gist (way)
+  WHERE place IN ('city', 'town') AND name IS NOT NULL;
+
+CREATE INDEX planet_osm_point_place
+  ON planet_osm_point USING gist (way)
+  WHERE place IS NOT NULL AND name IS NOT NULL;
+
+CREATE INDEX planet_osm_point_name
+  ON planet_osm_point USING gist (way)
+  WHERE name IS NOT NULL;
```

Comment:
This index is 653MB and only used by `text-point`, so I'm not this is worth the additional memory cache pressure.